qiskit_ibm_runtime/ibm_estimator.py

```diff
@@ -0,0 +1,141 @@
+        (a list of :class:`~qiskit.circuit.QuantumCircuit`))
+
+    * observables: a list of :class:`~qiskit.quantum_info.SparsePauliOp`
+
+    * parameters: a list of parameters of the quantum circuits.
+        (:class:`~qiskit.circuit.parametertable.ParameterView` or
+        a list of :class:`~qiskit.circuit.Parameter`) specifying the order
+        in which parameter values will be bound.
+
+    The :class:`qiskit_ibm_runtime.sessions.EstimatorSession` instance can be called repeatedly
+    with the following parameters to estimate expectation values.
+
+    * circuit_indices: A list of circuit indices.
+
+    * observable_indices: A list of observable indices.
+
+    * parameter_values: Concrete parameters to be bound.
+
+    All the above lists should be of the same length.
+
+    Example::
+
+        from qiskit.circuit.library import RealAmplitudes
+        from qiskit.quantum_info import SparsePauliOp
+
+        from qiskit_ibm_runtime import IBMRuntimeService, IBMEstimator
+
+        service = IBMRuntimeService(auth="cloud")
+        estimator_factory = IBMEstimator(service=service, backend="ibmq_qasm_simulator")
+
+        psi1 = RealAmplitudes(num_qubits=2, reps=2)
+        psi2 = RealAmplitudes(num_qubits=2, reps=3)
+
+        params1 = psi1.parameters
+        params2 = psi2.parameters
+
+        H1 = SparsePauliOp.from_list([("II", 1), ("IZ", 2), ("XI", 3)])
+        H2 = SparsePauliOp.from_list([("IZ", 1)])
+        H3 = SparsePauliOp.from_list([("ZI", 1), ("ZZ", 1)])
+
+        with estimator_factory([psi1, psi2], [H1, H2, H3], [params1, params2]) as estimator:
+            theta1 = [0, 1, 1, 2, 3, 5]
+            theta2 = [0, 1, 1, 2, 3, 5, 8, 13]
+            theta3 = [1, 2, 3, 4, 5, 6]
+
+            # calculate [ <psi1(theta1)|H1|psi1(theta1)> ]
+            psi1_H1_result = estimator([0], [0], [theta1])
+            print(psi1_H1_result)
+
+            # calculate [ <psi1(theta1)|H2|psi1(theta1)>, <psi1(theta1)|H3|psi1(theta1)> ]
+            psi1_H23_result = estimator([0, 0], [1, 2], [theta1]*2)
+            print(psi1_H23_result)
+
+            # calculate [ <psi2(theta2)|H2|psi2(theta2)> ]
+            psi2_H2_result = estimator([1], [1], [theta2])
+            print(psi2_H2_result)
+
+            # calculate [ <psi1(theta1)|H1|psi1(theta1)>, <psi1(theta3)|H1|psi1(theta3)> ]
+            psi1_H1_result2 = estimator([0, 0], [0, 0], [theta1, theta3])
+            print(psi1_H1_result2)
+
+            # calculate [ <psi1(theta1)|H1|psi1(theta1)>,
+            #             <psi2(theta2)|H2|psi2(theta2)>,
+            #             <psi1(theta3)|H3|psi1(theta3)> ]
+            psi12_H23_result = estimator([0, 0, 0], [0, 1, 2], [theta1, theta2, theta3])
+            print(psi12_H23_result)
+    """
+
+    def __call__(  # type: ignore[override]
+        self,
+        circuits: Iterable[QuantumCircuit],
+        observables: Iterable[SparsePauliOp],
+        parameters: Optional[Iterable[Iterable[Parameter]]] = None,
```

Comment:
also should be `circuit_indices`/`observable_indices`/`parameter_values`?

Reply:
No this is the actual circuits, observables and parameters.